src/utils.test.js

```diff
@@ -0,0 +1,22 @@
+import { expect, it, describe } from "vitest";
+import { newDate, chomp /* mapChomp, Classifier  */ } from "./utils.js";
+
+describe("newDate", () => {
+  it("parses DD/MM/YYYY dates correctly", () => {
+    let d = newDate("10/12/1999");
+    expect(d).toEqual(new Date(1999, 12, 10));
+  });
+  ["10/12/19", "10/12/1999/2", "12/14/1999"].map((d) =>
```

Comment:
Tests should have as little logic as possible. I'd rather we had three almost-identical tests than have this mapping

Reply:
it makes it tedious to add more breaking cases, i expect we may find more of them, or may want to do more tests on them.

Reply:
Alright, maybe add this mapping to a new `it` scope so that it's clear that these are invalid date strings